face/src/main/java/io/blockv/face/client/FaceManager.kt

```diff
@@ -6,9 +16,26 @@ import android.view.ViewGroup
 import io.blockv.common.model.Face
 import io.blockv.common.model.Vatom
 import io.blockv.common.util.Callable
+import io.blockv.face.client.FaceManager.EmbeddedProcedure.*
 
 interface FaceManager {
 
+  /**
+   * EmbeddedProcedure enum contains a set of default face selection procedures.
+   *
+   * By default these procedures will only look at native faces that have a platform value 'Generic'
+   * or 'Android'.
+   *
+   * Faces with a platform value of 'Android' will be selected over 'Generic' in the case where there are
+   * multiple faces per view mode.
+   *
+   * @see ICON - Selects a Face model that is in icon mode, if none is available it returns null.
```

Comment:
ICON - Selects a Face model in icon mode, returns `null`  if none are available.